### PR TITLE
Pre-need Self Service: Supporting Files - file type drop-down selections are changing when multiple files are uploaded

### PR DESCRIPTION
**Pull request by jordanBoardmanCodes**
## Summary

Issue:

If a user has uploaded multiple files on the Supporting files page, making a selection for "What kind of file is this?" can affect the same field for other uploaded documents.

## Related issue(s)

Jira ticket - https://jira.devops.va.gov/browse/MBMS-60395

## Screenshots

https://github.com/department-of-veterans-affairs/vets-website/assets/155088633/6c83d9d0-02e0-4f55-ad5e-f3a26ceff528

https://github.com/department-of-veterans-affairs/vets-website/assets/155088633/721c5e22-f112-4bed-8d35-12792ba5c65a

## What areas of the site does it impact?

Pre-Need & Pre-Need-Integration

## Acceptance criteria

Screenshot 2024-07-09 at 3 35 45 PM

### Quality Assurance & Testing

- [x] POR/UI/UX Review
- [x] Internal Dev Approval
- [ ] External Dev Approval
- [x] QA
- [x] Unit Tests Passing

### CODE REVIEW

File: src/applications/pre-need-integration/config/form.jsx [hunk 1]
```diff
@@ -6,7 +6,8 @@ import environment from 'platform/utilities/environment';
 import preSubmitInfo from 'platform/forms/preSubmitInfo';
 import { VA_FORM_IDS } from 'platform/forms/constants';
 
-import fileUploadUI from 'platform/forms-system/src/js/definitions/file';
+// import fileUploadUI from 'platform/forms-system/src/js/definitions/file';
+import { fileUploadUi } from '../../pre-need/utils/upload';
```
Comment (adam-antonioli) [inline src/applications/pre-need-integration/config/form.jsx:10]: Check that linting warning for this line. Don't think we want to import from a different project, so I'd copy upload.js to pre-need-integration too

Comment (jordanBoardmanCodes) [reply src/applications/pre-need-integration/config/form.jsx]: Should be good to go now.

**Review by adam-antonioli — DISMISSED** [review]

File: src/applications/pre-need/config/form.jsx [hunk 2]
```diff
@@ -368,17 +368,12 @@ const formConfig = {
                       'ui:title': 'File name',
                     },
                   })
-                : fileUploadUI('Select files to upload', {
-                    buttonText: 'Upload file',
-                    addAnotherLabel: 'Upload another file',
+                : fileUploadUi({
                     fileUploadUrl: environment.isProduction()
```
Comment (adam-antonioli) [inline src/applications/pre-need/config/form.jsx:372]: This is missing from the integration version. Is there a reason you took it out there?

File: src/applications/pre-need/config/form.jsx [hunk 3]
```diff
@@ -368,17 +368,12 @@ const formConfig = {
...
                       ? `${environment.API_URL}/v0/preneeds/preneed_attachments`
                       : `${
                           environment.API_URL
                         }/simple_forms_api/v1/simple_forms/submit_supporting_documents`,
-                    fileTypes: ['pdf'],
-                    maxSize: 15728640,
-                    hideLabelText: true,
                     createPayload: !environment.isProduction()
```
Comment (adam-antonioli) [inline src/applications/pre-need/config/form.jsx:377]: This is missing from the integration version. Is there a reason you took it out there?

Comment (jordanBoardmanCodes) [reply src/applications/pre-need/config/form.jsx:377]: Yes, those properties are now being handled in the new `upload,js` component.

File: src/applications/pre-need/config/form.jsx (hunk 2 above)
Comment (jordanBoardmanCodes) [reply src/applications/pre-need/config/form.jsx:372]: Yes, those properties are now being handled in the new `upload,js` component as well.

**Review by delli-gatti-michael — APPROVED** [review]

**Review by ksantiagoBAH — APPROVED** [review]

**Pull request merged**